packages/webapp-libs/webapp-tenants/src/components/tenantMembersList/membershipEntry/membershipEntry.component.tsx

```diff
@@ -11,36 +12,71 @@ import {
   DropdownMenuSubTrigger,
   DropdownMenuTrigger,
 } from '@sb/webapp-core/components/dropdownMenu';
+import { Skeleton as SkeletonComponent } from '@sb/webapp-core/components/skeleton';
 import { TableCell, TableRow } from '@sb/webapp-core/components/table';
+import { useToast } from '@sb/webapp-core/toast';
 import { GripHorizontal, Hourglass, UserCheck } from 'lucide-react';
 import { indexBy, prop, trim } from 'ramda';
 import { useMemo } from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, useIntl } from 'react-intl';
 
 import { useTenantRoles } from '../../../hooks/useTenantRoles';
+import { useCurrentTenant } from '../../../providers';
+import { updateTenantMembershipMutation } from './membershipEntry.graphql';
 
 export type MembershipEntryProps = {
   className?: string;
   membership: TenantMembershipType;
+  refetch?: () => void;
 };
 
-export const MembershipEntry = ({ membership, className }: MembershipEntryProps) => {
+export const MembershipEntry = ({ membership, className, refetch }: MembershipEntryProps) => {
+  const { data: currentTenant } = useCurrentTenant();
+  const { toast } = useToast();
   const { getRoleTranslation } = useTenantRoles();
+  const intl = useIntl();
+
+  const successMessage = intl.formatMessage({
+    id: 'Membership Entry / UpdateRole / Success message',
+    defaultMessage: '🎉 The user role was updated successfully!',
+  });
+
+  const [commitUpdateMutation, { loading }] = useMutation(updateTenantMembershipMutation, {
+    onCompleted: () => {
```

Comment:
Nice to have the success message but would be good to also show user if something went wrong ;)